extension-manifest-v3/src/background/reporting/index.js

```diff
@@ -137,11 +137,13 @@ if (__PLATFORM__ === 'firefox') {
       if (event === 'observed') {
         return;
       }
-      const request = Request.fromRawDetails({
+
+      const request = Request.fromRequestDetails({
         url: state.url,
-        sourceUrl: state.tabUrl,
+        originUrl: state.tabUrl,
```

Comment:
not a comment to a name change but in general - not sure if we should relay in `state.tabUrl` as it comes from `page-store` of the wtm/reporting webrequest-pipeline and we know that this one was a subject of same kind of state leakage. Yes, the latest version should work better, but maybe we should find another mechanism that will be consistent between all features.

Reply:
I think it is not related to this PR. Please "save the case for later" in your mind, or make an issue about it to not forget the problem.